packages/apple-pay-integration/src/apple-pay-payment-strategy.ts

```diff
@@ -235,4 +248,27 @@ export default class ApplePayPaymentStrategy implements PaymentStrategy {
             );
         }
     }
+
+    private async _getDataCollector() {
+        const state = await this._paymentIntegrationService.loadPaymentMethod(
```

Comment:
It could be better to load braintree in initialization method

Reply:
Moved load braintree in initialization method